desktop/src/main/java/bisq/desktop/components/list/FilterBox.java

```diff
@@ -0,0 +1,56 @@
+package bisq.desktop.components.list;
```

Comment:
Copyright notice is missing.

Reply:
and on a couple of other places. I've adapted the Codacy checks so it should be checked from now on.

Reply:
[Fixed](https://github.com/bisq-network/bisq/pull/5963/commits/4268d5e451036d3c2f041f897d147dd95286499c)